src/main/java/org/fcrepo/spec/testsuite/crud/Container.java

```diff
@@ -729,4 +732,571 @@ public void ldpDirectContainerMayAllowLdpIsMemberRelationToBeUpdatedByPatch() {
                                                    LDP_IS_MEMBER_OF_RELATION_PREDICATE, isMemberOfPredicate1, false);
 
     }
+
+
+    /**
+     * 3.1.3-A
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustAllowMembershipConstantURIToBeSet() {
+        setupTest("3.1.3-A",
+                  "Implementations " +
+                  "MUST allow the indirect container's membership constant URI to be set via the " +
+                  "ldp:membershipResource property of the content RDF on container creation.",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String membershipResource = getLocation(doPost(uri));
+
+        final String body = "@prefix ldp: <http://www.w3.org/ns/ldp#> .\n"
+                            + "<> ldp:membershipResource <" + membershipResource + "> ;";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        confirmPresenceOrAbsenceOfTripleInResponse(doGet(containerResource), containerResource,
+                                                   LDP_MEMBERSHIP_RESOURCE_PREDICATE, membershipResource, true);
+    }
+
+    /**
+     * 3.1.3-B
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustSetLdpMembershipResourceByDefault() {
+        setupTest("3.1.3-B",
+                  "Implementations MUST set the indirect container's ldp:membershipResource by default when" +
+                  " not specified on creation.",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String body = "";
+        final Response directContainer = createDirectContainer(uri, body);
+        final String directContainerResource = getLocation(directContainer);
+        final String responseBody = doGet(getLocation(directContainer)).getBody().asString();
+        try (CloseableDataset dataset = parseTriples(IOUtils.toInputStream(responseBody))) {
```

Comment:
Why not use `confirmPresenceOrAbsenceOfTripleInResponse` here?

Reply:
Because I don't know what the object will be.